Packs/RDPCacheHunting/Playbooks/playbook-RDP_Bitmap_Cache_-_Detect_and_Hunt.yml

```diff
@@ -0,0 +1,1525 @@
+      type: regular
+      iscommand: true
+      brand: Rasterize
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      EntryID:
+        complex:
+          root: InfoFile
+          filters:
+          - - operator: containsString
+              left:
+                value:
+                  simple: InfoFile.Name
+                iscontext: true
+              right:
+                value:
+                  simple: RDPCacheImage
+              ignorecase: true
+          - - operator: containsString
+              left:
+                value:
+                  simple: InfoFile.Extension
+                iscontext: true
+              right:
+                value:
+                  simple: pdf
+              ignorecase: true
+          accessor: EntryID
+      file_name:
+        simple: RDPCacheImage
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1060,
+          "y": 1070
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "47":
+    id: "47"
+    taskid: baef2a3c-06f5-454a-87d1-ce1a06e0b83f
+    type: regular
+    task:
+      id: baef2a3c-06f5-454a-87d1-ce1a06e0b83f
+      version: -1
+      name: Convert pdf to image
+      description: Converts an image file to a PDF file.
+      script: Rasterize|||rasterize-image
+      type: regular
+      iscommand: true
+      brand: Rasterize
+    nexttasks:
+      '#none#':
+      - "46"
+    scriptarguments:
+      EntryID:
+        complex:
+          root: File
+          filters:
+          - - operator: containsString
+              left:
+                value:
+                  simple: File.Name
+                iscontext: true
+              right:
+                value:
+                  simple: original
+              ignorecase: true
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: File.Extension
+                iscontext: true
+              right:
+                value:
+                  simple: png
+              ignorecase: true
+          accessor: EntryID
+      file_name:
+        simple: RDPCacheImage
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1060,
+          "y": 880
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+system: true
+view: |-
+  {
+    "linkLabelsPosition": {
+      "31_10_yes": 0.35,
+      "31_32_#default#": 0.1,
+      "33_35_yes": 0.48
+    },
+    "paper": {
+      "dimensions": {
+        "height": 3460,
+        "width": 2530,
+        "x": -840,
+        "y": -1365
+      }
+    }
+  }
+inputs:
+- key: ShouldCollectRDPCache
```

Comment:
I see you also used the Retrieve File from Endpoint - Generic V3 playbook.
That means that we need to document in the playbook's readme and the input description that we can collect using x,y,z.